app/javascript/packs/CommentTextArea/CommentTextArea.jsx

```diff
@@ -36,13 +42,33 @@ const handleImageSuccess = (textAreaRef) => {
     editableBodyElement.selectionStart =
       selectionStart + markdownImageLink.length;
     editableBodyElement.selectionEnd = editableBodyElement.selectionStart;
+
+    // Dispatching a new event so that linkstate, https://github.com/developit/linkstate,
+    // the function used to create the onChange prop gets called correctly.
+    editableBodyElement.dispatchEvent(new Event('input'));
```

Comment:
The comment area doesn't use `linkstate` so I think it should be safe to remove this? I tried locally and it seems to work as expected, but maybe I'm missing something!

Reply:
Removed as it is not required.